lib/glucose-get-last.js

```diff
@@ -65,6 +65,7 @@ var getLastGlucose = function (data) {
     return {
         delta: Math.round( last_delta * 100 ) / 100
         , glucose: Math.round( now.glucose * 100 ) / 100
+        , noise: now.noise
```

Comment:
Should we round it to the appropriate precision to make sure we don't have floating point math fun like 3.00000000000001?

Reply:
i guess we could, probably wouldn't hurt, but at the moment, i only saw integer values for this field, but let me fix that.

Reply:
fixed